tasks/push-binaries.js

```diff
@@ -31,13 +31,23 @@ const platformOsMap = {
 /**
  * Push all compiler binaries built on this OS after syncing to origin
  **/
-(async function () {
+async function main() {
   const osName = platformOsMap[process.platform];
-  const compilerBinaries = path.join('packages', `google-closure-compiler-${osName}`, 'compiler*')
-  execOrDie(`git add ${compilerBinaries}`);
+  const compilerBinaryPaths = path.join('packages', `google-closure-compiler-*`, 'compiler*')
+  console.log(compilerBinaryPaths);
+  execOrDie(`git config --global user.name "${process.env.GITHUB_ACTOR}"`);
+  execOrDie(`git config --global user.email "${process.env.GITHUB_ACTOR}@users.noreply.github.com"`);
+  execOrDie('git branch');
+  execOrDie('git status');
+  execOrDie('git log -n5');
+  execOrDie(`git add ${compilerBinaryPaths}`);
   execOrDie(`git commit -m "Push new ${osName} compiler binaries"`);
+  execOrDie('git diff');
+  execOrDie('git status');
   execOrDie('git pull --rebase');
   execOrDie('git push');
-})();
+}
+
+main();
```

Comment:
Any reason to not use the existing immediate execution?

Reply:
GH actions didn't seem happy with it. I tried debugging to see if it was a typo, but decided to change the call to proceed. Happy to switch it back if you can spot the problem.

![image](https://user-images.githubusercontent.com/26553114/89799503-8f0f1c00-dafb-11ea-9c5d-3f6f7abbec8a.png)

Reply:
Wild. No issue from my perspective to change.

Reply:
Yeah, mysterious. I'll investigate this in a follow up after this is merged.